MEMBER_TRAVEL_FUND.md

```diff
@@ -6,12 +6,15 @@ and the Node.js Foundation projects.
 
 ## Restrictions
 
-* Candidates must be an individual member of the Node.js Foundation.
+* Candidates must be an individual member of the Node.js Foundation. The membership
+  status can be checked on [the foundation website][].
+* Employees of a corporate member of the Node.js Foundation cannot request for
```

Comment:
What about employees of a corporate member who can't get their company to agree to expense the travel?

Reply:
In the past, we've expressed an expectation that foundation members would support relevant Node.js travel for their employees rather than having employees make travel requests. I don't think we should make a hard rule about it though. As with other people (with relevant employment) making travel requests, there is an expectation that employees of foundation members have asked their employer first and only come to us after they could not get their employer to pay for their travel.

Reply:
This may need discussions from the board. The travel fund comes from donations from the member companies so it's up to them to decide whether it's fair to spend that travel fund to sponsor an employee from a particular member company.